docs/task-attachment-list.component.md

```diff
@@ -24,6 +25,30 @@ Displays attached documents on a specified task
     (attachmentClick)="YOUR_HANDLER">
 </adf-task-attachment-list>
 ```
+### How to Add Drag and Drop Functionality
+
+If we want user to be able to upload attachments for empty lists, We can wrap our component with upload drag area component. In that case, We should also pass a custom *no content template* as shown below with our component urging the user to drag files to upload whenever the list is empty.
+
+```html
+<adf-upload-drag-area
+    [parentId]="YOUR_TASK_ID"
+    [showNotificationBar]="BOOLEAN">
+    <adf-task-attachment-list  
+        [taskId]="YOUR_TASK_ID"
+        (attachmentClick)="YOUR_HANDLER">
+            <div adf-empty-list> //no content template
+                <adf-empty-list>
+                    <div adf-empty-list-header>{{This List is empty}}</div>
+                    <div adf-empty-list-body>{{Drag and drop to upload}}</div>
+                    <div adf-empty-list-footer>
+                        <img [src]="Your custom image URL"></div> 
+                </adf-empty-list>
+            </div>
+    </adf-task-attachment-list>
+</adf-upload-drag-area>
+```
```

Comment:
can we attach an image of how the default template looks like and one related to the custom drag&drop template?